src/Console/Commands/stubs/auth/statamic_auth_tables.php.stub

```diff
@@ -15,18 +15,19 @@ class StatamicAuthTables extends Migration
     {
         USERS_TABLE_REPLACE
         Schema::create('ROLE_USER_TABLE', function (Blueprint $table) {
-            $table->increments('id');
-            $table->integer('user_id');
+            $table->id('id');
+            $table->foreignId('user_id')->constrained('USERS_TABLE')->cascadeOnDelete();
             $table->string('role_id');
         });
 
         Schema::create('GROUP_USER_TABLE', function (Blueprint $table) {
-            $table->increments('id');
-            $table->integer('user_id');
+            $table->id('id');
+            $table->foreignId('user_id')->constrained('USERS_TABLE')->cascadeOnDelete();
             $table->string('group_id');
         });
 
         Schema::create('password_activations', function (Blueprint $table) {
+            $table->id('id');
```

Comment:
The core devs didnt have this in originally - why is it needed?

Reply:
Some databases require primary id's on tables for duplication - typically in clusters.